### ISSUE TITLE
Explanations don't match the variable names in the code in https://js.tensorflow.org/tutorials/webcam-transfer-learning.html

### ISSUE THREAD

**Issue by ToonTalk** [tensorflow/tfjs-website#121]
The explanation for

```
const act = mobilenet.predict(img);
const predictions = model.predict(act);
```
Refers to activation and logits instead of act and predictions.

**Comment by tafsiri** [comment]
Thanks for reporting this @ToonTalk. A fix is on the way.
